roles/common/tasks/azure-storage-configuration.yml

```
@@ -118,7 +118,7 @@
 
 - name: Add azure connection string to azure settings
   set_fact:
-    azure_connection_string_dict: "{'AZURE_CONNECTION_STRING': '{{ _custom_azure_configuration['resources'][0]['data']['azure-connection-string'] | b64decode }}' }"
+    azure_connection_string_dict: {'AZURE_CONNECTION_STRING': '{{ _custom_azure_configuration['resources'][0]['data']['azure-connection-string'] | b64decode }}'}
```

We could also update the set_fact tasks to use a better yaml dict format but this isn't blocking this PR to be merged.
Feel free to update it if you want

```suggestion
    azure_connection_string_dict:
      AZURE_CONNECTION_STRING: '{{ _custom_azure_configuration['resources'][0]['data']['azure-connection-string'] | b64decode }}'
```

---

@dsavineau thank you for the review, I ammended this suggestion.